helix-ui/src/main/java/org/apache/helix/ui/api/IdealStateSpec.java

```diff
@@ -0,0 +1,94 @@
+package org.apache.helix.ui.api;
+
+import org.apache.helix.model.IdealState;
+
+import java.util.List;
+
+public class IdealStateSpec {
```

Comment:
jsonschema2pojo (or similar) would be useful for classes like this, but not a big deal.